src/main/java/org/folio/services/validator/engine/ValidationEngineServiceImpl.java

```diff
@@ -1,11 +1,6 @@
 package org.folio.services.validator.engine;
 
-import io.vertx.core.AsyncResult;
-import io.vertx.core.CompositeFuture;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.MultiMap;
-import io.vertx.core.Vertx;
+import io.vertx.core.*;
 import io.vertx.core.http.CaseInsensitiveHeaders;
 import io.vertx.core.http.HttpClient;
```

Comment:
Vert.x 3.6.2 deprecated HttpClient request/response methods: https://github.com/vert-x3/wiki/wiki/3.6.2-Deprecations
Can you use WebClient instead?